docs/dashboards/data-quality-dashboard.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[data-quality-dash]]
+= Data Quality dashboard
+
+The Data Quality dashboard shows you whether your data is correctly mapped to the https://www.elastic.co/guide/en/ecs/current/ecs-reference.html[Elastic Common Schema] (ECS). Successful {ref}/mapping.html[mapping] enables you to search, visualize, and interact with your data throughout {es-sec} and {kib}.
+
+image::images/data-qual-dash.png[The Data Quality dashboard]
+
+You can use the Data Quality dashboard to:
+
+* Check one or multiple indices for unsuccessful mappings, to help you identify problems (the indices used by {es-sec} appear by default).
+* View detailed information about the fields in checked indices.
+* Track unsuccessful mappings by creating a case or Markdown report based on data quality results.
+
+NOTE: The Data Quality dashboard does not show data from cold or frozen {ref}/data-tiers.html[data tiers]. It also does not display data from remote clusters using cross-cluster search — to view data from another cluster, log in to that cluster's {kib} instance.
```

Comment:
Edit suggestions for tone, and a semicolon would prob be more appropriate to join clauses like these. I think a dash is better for interjecting "extra" or maybe unexpected information, not just transitioning to the next step in a procedure.
```suggestion
NOTE: The Data Quality dashboard doesn't show data from cold or frozen {ref}/data-tiers.html[data tiers]. It also doesn't display data from remote clusters using cross-cluster search; to view data from another cluster, log in to that cluster's {kib} instance.
```